taxi/models.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from taxi_service import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+    country = models.CharField(max_length=100)
+
+    def __str__(self):
+        return f"{self.name} {self.country}"
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=20, unique=True)
+
+    def __str__(self):
+        return f"{self.username} {self.first_name} {self.last_name} {self.license_number}"
```

Comment:
That's a bit too much information - since username and license number are unique it would be enough to display only those in string presentation